package.json

```diff
@@ -6,6 +6,7 @@
   "scripts": {
     "build": "yarn plugin-helpers build",
     "plugin-helpers": "node ../../scripts/plugin_helpers",
+    "test": "../../node_modules/.bin/jest --config ./test/jest.config.js",
```

Comment:
similar to below will yarn test:jest not capture these tests?

Reply:
in queryEnhancements yarn test:jest is not defined

Reply:
yeah i mean like like if i was in the root repo and ran like

yarn test:jest {pathtotest} does it work?